yjit/src/disasm.rs

```diff
@@ -158,24 +158,25 @@ pub fn disasm_addr_range(cb: &CodeBlock, start_addr: usize, end_addr: usize) ->
     #[cfg(test)]
     let start_addr = 0;
     let insns = cs.disasm_all(code_slice, start_addr as u64).unwrap();
+    let color = get_option!(color);
 
     // For each instruction in this block
     for insn in insns.as_ref() {
         // Comments for this block
         if let Some(comment_list) = cb.comments_at(insn.address() as usize) {
             for comment in comment_list {
                 if cb.outlined {
-                    write!(&mut out, "\x1b[34m").unwrap(); // Make outlined code blue
+                    write!(&mut out, "{}", color.blue_begin).unwrap(); // Make outlined code blue
                 }
-                writeln!(&mut out, "  \x1b[1m# {comment}\x1b[22m").unwrap(); // Make comments bold
+                writeln!(&mut out, "  {}# {comment}{}", color.bold_begin, color.bold_end).unwrap(); // Make comments bold
```

Comment:
```suggestion
                writeln!(&mut out, "  {color.bold_begin}# {comment}{color.bold_end}").unwrap(); // Make comments bold
```

It seems weird to me to mix the use of different interpolation styles (`"{comment}"` vs `"{}", color.bold_begin`) in the same line. The same applies to other uses of `color`.

Reply:
Interpolation only works for simple reference to locals like `"{comment}"` unfortunately, so "{color.bold_end}" doesn't build. Could switch to all "{}", though.

Reply:
Oh sorry, I tested it using a release build, so it wasn't tested correctly. Never mind then :pray: